plugins/slack_webhooks/lib/samson_slack_webhooks/slack_webhooks_service.rb

```diff
@@ -0,0 +1,49 @@
+module SamsonSlackWebhooks
+  class SlackWebhooksService
+    # users in the format jquery mentions input needs
+    # see _notify_buddy_box.html.erb
+    def users
+      Rails.cache.fetch(:slack_users, expires_in: 5.minutes, race_condition_ttl: 5) do
+        if !slack_api_token
+          Rails.logger.error('Set the SLACK_API_TOKEN env variable to enabled user mention autocomplete.')
+          []
```

Comment:
sorry to nitpick, in my comment before I meant something like

```ruby
def users
  unless slack_api_token
    Rails.logger.error('Set the SLACK_API_TOKEN env variable to enabled user mention autocomplete.')
    return []
  end
  body = JSON.parse(Faraday.post("https://slack.com/api/users.list", token: slack_api_token).body)
  if body['ok']
    ...
end
```

due to the early return there's no need to indent the rest of the function in an `else` block (and I think we can get away with not using `begin` / `end` here as well